_posts/2022-04-28-introducing-uswds-3-0.md

```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
 
 [USWDS browser support (2% rule)](https://designsystem.digital.gov/documentation/developers/#browser-support) →
 
-[See current analytics from analytics.usa.gov](https://analytics.usa.gov/) → 
+[See current analytics from analytics.usa.gov](https://analytics.usa.gov/)
 
 ### Support some Javascript-required functionality 
 
 We believe that JavaScript-disabled environments fall below our 2% support threshold, and that we may require JavaScript for some design system functionality. While we have not made any changes to our codebase that affect how we use JavaScript with our components, we may make changes that require JavaScript for some functionality as we move forward. When we do make any changes like this, we'll be sure to note it in our release notes.
 
 ### Support USWDS 2 with maintenance and security patches for one year
 
-In addition to supporting active development on USWDS 3, We will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
+In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
```

Comment:
```suggestion
In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/releases/tag/v2.13.3).
```

Let's link to the release, not the discussion

Reply:
Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)